### DISCUSSION TITLE
Summierte Energie aus zwei Wechselrichtern

### DISCUSSION THREAD

**Discussion by akitzing** [tbnobody/OpenDTU#1506] [Q&A]
Ich habe zwei verschiedene Wechselrichter per Betteri-Kabel miteinander verbunden. 
Der HM-800 ist mit der Steckdose verbunden. Mir werden nur die Details der Panels und der damit umgewandelte Strom (Phase 1) angezeigt.  
Gibt es eine Möglichkeit die summierten Strom bzw. den vom anderen Wechselrichter eingehenden Strom zu messen und anzuzeigen?

**Comment by Omega13x** [comment]
Ja, du musst den zweiten WR auch in der Konfiguration anlegen. Dann wird im Dashboard oben der Gesamtertrag und die Gesamtleistung angezeigt.

![image](https://github.com/tbnobody/OpenDTU/assets/123419851/766b65e0-ba2c-4ab3-9466-bf21ecd75c28)

![image](https://github.com/tbnobody/OpenDTU/assets/123419851/2b489618-04f5-4b97-91b1-33bed73c8e44)

**Reply by akitzing** [reply]
Es handelt sich leider beim Zweiten um einen nicht opendtu kompatiblen Wechselrichter.

**Reply by Omega13x** [reply]
Dann hast du Pech. Das Betterie-Kabel ist nur Durchgeschleift. Da wird nichts gemessen.

**Reply by akitzing** [reply]
Schade, aber Danke für die schnelle Erkenntnis.
